CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### master
 
+### 5.9.0-alpha.4 - March 24, 2021
+ - Testing automated release process [#1245](https://github.com/mapbox/mapbox-java/pull/1245)
+
```

Comment:
Should we remove this?

Reply:
idk.. making a process here :). when do we decide not to add a changelog for a release?

Reply:
Are we ultimately releasing? 🤔

Reply:
actually yeah, removed it. considering that we're not making an automatic publish we don't need to cut a release